### PR TITLE
Format code - aws folder

### PR DESCRIPTION
**Pull request by vivekmore**
format code as per project's .editorconfig

### CODE REVIEW

File: generators/aws/lib/s3.js [hunk 1]
```diff
@@ -107,8 +107,8 @@ var findWarFilename = function findWarFilename(buildFolder, callback) {
             error(err, callback);
         }
         files.filter(function (file) {
-            return file.substr(-FILE_EXTENSION.length) === FILE_EXTENSION;
-        })
+                return file.substr(-FILE_EXTENSION.length) === FILE_EXTENSION;
```
Comment (deepu105) [inline generators/aws/lib/s3.js]: is this correct?

Comment (deepu105) [inline generators/aws/lib/s3.js]: the original indent seems correct to me

Comment (deepu105) [inline generators/aws/lib/s3.js]: im correcting this

Comment (vivekmore) [inline generators/aws/lib/s3.js]: Good catch !
Yes the original looks correct.
I don't know why Idea would format it that way.
I'll see if that is a bug in Idea.

**Pull request merged**